process/logsevents/nftsProcessor.go

```diff
@@ -46,30 +50,44 @@ func (np *nftsProcessor) processEvent(args *argsProcessEvent) (string, bool) {
 		return "", false
 	}
 
-	sender := args.event.GetAddress()
-	if np.shardCoordinator.ComputeId(sender) == np.shardCoordinator.SelfId() {
-		np.processNFTEventOnSender(args.event, args.accounts, args.tokens, args.timestamp, args.tagsCount)
-	}
-
 	// topics contains:
 	// [0] -- token identifier
 	// [1] -- nonce of the NFT (bytes)
-	// [2] -- receiver NFT address -- in case of NFTTransfer OR ESDT token data in case of NFTCreate
+	// [2] -- value
+	// [3] -- receiver NFT address -- in case of NFTTransfer OR ESDT token data in case of NFTCreate
 	topics := args.event.GetTopics()
-	token := string(topics[0])
 	nonceBig := big.NewInt(0).SetBytes(topics[1])
+	if nonceBig.Uint64() == 0 {
+		// is a fungible token should return
```

Comment:
`// this is a fungible token so we should return`

Reply:
done.